docs/src/modules/ROOT/partials/concepts-intro.adoc

```diff
@@ -44,6 +44,60 @@ You create a continuous incoming stream of events with the xref:java:http-endpoi
+
+=== Workflows
+
+image:ROOT:workflow.png[Workflows,width=100,float=left] _Workflows_ enable the developer to implement long-running, multi-step business processes while focusing exclusively on domain and business logic. Technical concerns such as delivery guarantees, scaling, error handling and recovery are managed by Akka. For more information see xref:java:workflows.adoc[Workflows].
+
+=== Entities
+
+image:ROOT:entity.png[Entities,width=100,float=left] _Entities_ are the core components of Akka and provide persistence and state management. They map to your https://martinfowler.com/bliki/DDD_Aggregate.html[_domain aggregates_, window="new"]. If you have a "Customer" domain aggregate, you almost certainly will have a `CustomerEntity` component to expose and manipulate it. This separation of concerns allows the domain object to remain purely business logic focused while the Entity handles runtime mechanics. Additionally, you may have other domain objects that are leafs of the domain aggregate. These do not need their own entity if they are just a leaf of the aggregate. An address is a good example.
+
+There are two types of entities in Akka. Their difference lies in how they internally function and are persisted.
+
+==== Key Value Entities
+
+image:ROOT:key-value-entity.png[Key Value Entities,width=100,float=left] _Key Value Entities_ are, as the name implies, an object that is stored and retrieved based on a key - an identifier of some sort. The value is the entire state of the object. Every write to a Key Value Entity persists the entire state of the object. Key Value Entities are similar in some ways to database records. They write and effectively lock the whole row. They still use an underlying event-based architecture so other components can subscribe to the stream of their updates. For more information see xref:java:key-value-entities.adoc[Key Value Entities].
+
+==== Event Sourced Entities
+
+image:ROOT:event-sourced-entity.png[Event Sourced Entities,width=100,float=left] _Event Sourced Entities_ persist events instead of state in the event xref:reference:glossary.adoc#journal[journal]   . The current state of the entity is derived from these events. Readers can access the event journal independently of the active entity instance to create read models, known as xref:java:views.adoc[_Views_], or to perform business actions based on the events via xref:java:consuming-producing.adoc[Consumers]. For more information, see xref:java:event-sourced-entities.adoc[Event Sourced Entities].
+
+=== Views
+
+image:ROOT:view.png[Views,width=100,float=left] _Views_ provide a way to materialize read only state from multiple entities based on a query. You can create views from Key Value Entities, Event Sourced Entities, and by subscribing to a topic. For more information see xref:java:views.adoc[Views].
```

Comment:
We should also mention that views can be created from workflow state change and from broker messages.